### DISCUSSION TITLE
How to scale the window decoration buttons?

### DISCUSSION THREAD

**Discussion by ghost** [slint-ui/slint#3792] [Q&A] [answered]
How to scale the window decoration buttons?

**Comment by ogoffart** [comment, marked as answer]
This really depends on the window manager,  and this is usually done by changing settings in the configuration of the window manager or OS

**Reply by ghost** [reply]
I thought that it would be managed by wm but the icons just looks like the program renders them by itself so I thought it is possible... Thanks!

**Reply by bjorn** [reply]
A screenshot might help to illustrate the problem. I'm not sure what you mean with "the icons just looks like the program renders them by itself". Do they look different from the buttons on other windows?

**Reply by ghost** [reply]
![Screenshot from 2023-10-31 14-55-54](https://github.com/slint-ui/slint/assets/97330399/3af6d444-4fd9-4ef5-9d85-5099dca5ce46)
It has different a style (a bit another top bar color and obviously buttons) so I decided that it renders it by itself...

**Reply by hunger** [reply]
Is this wayland?

All the UI libraries render the windows themselves there -- not that programmers can necessarily influence that without drawing everything themselves :-/

It's basically impossible to get consistent window decoration on wayland.

**Reply by ghost** [reply]
Yeah it is gnome with wayland
